### ISSUE TITLE
sync xmLHttpRequest not work 

### ISSUE THREAD

**Issue by zhangolve** [requestly/requestly#1941]
Hello,

I have been using this extension for several years, but after upgrading to the latest version, my website is not working very well.

On my website, I use jquery.ajax, and sometimes I need to use synchronous AJAX requests. An example of my code is:

```
$.ajax({
    url: "testOperation.php",
    async: false,
    data: compareHeure,
    success: function (data, statusRequest) {
        // some code
    }
});
```

However, it seems that when I enable this extension in my Chrome browser, the behavior remains asynchronous even though I have set async: false. Do you think this is a bug that needs to be fixed?

**Comment by wrongsahil** [comment]
Hey @zhangolve . Is you website breaking due to this? or is it just the behaviour of XHR being async?

**Comment by zhangolve** [comment]
Yes , My website is breaking due to this issue because I am using xhr sync logic to fetch data. @wrongsahil
